src/python/deepgnn/graph_engine/snark/client.py

```diff
@@ -152,6 +152,7 @@ def __init__(
         storage_type: PartitionStorageType = PartitionStorageType.memory,
         config_path: str = "",
         stream: bool = False,
+        enable_threadpool: bool = False,
```

Comment:
This is only for server so can we remove it from local ge?

Reply:
My understanding is that both remote and local GE will use this feature, right? Do we have any concern to use it in local GE?

Reply:
I see I think argument hint is wrong then, https://github.com/microsoft/DeepGNN/pull/93/files#:~:text=help%3D%22-,Whether%20or%20not%20to%20enable%20server%20thread%20pool,-.%22%2C

Reply:
Updated, thanks a lot!